cypress/features/visual/checkboxValidations.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Checkbox validations component
+  I want check that all examples of Checkbox component - validation story render correctly
+
+  @positive
+  @applitools
+  Scenario: Check that all examples of Checkbox component - validation story render correctly
+    When I open "Experimental Checkbox" component page validations in iframe
```

Comment:
I see on the applitools screen that you open `noiFrame` state
```suggestion
    When I open "Experimental Checkbox" component page validations in noiFrame
```

Reply:
updated